SL/Mailer.pm

```diff
@@ -51,13 +51,22 @@ sub apisend {
   $data->{to}->[0]->{email} = $self->{to};
 
   if ($self->{cc}){
-     $data->{cc}->[0]->{name} = $self->{cc};
-     $data->{cc}->[0]->{email} = $self->{cc};
+     @cc = split /,/, $self->{cc};
+     my $i = 0;
+     for (@cc){
```

Comment:
doesn't work in case of spaces, I added the emails like philipp.kroll@gmail.com, pkroll@runmyaccounts.com.
Is there something like a trim we can do before the split?

Reply:
Yes, we can remove all white space before split. I do this change and update.